server.js

```diff
@@ -201,6 +201,20 @@ server.post('/invitation', jsonParser, function(req, res) {
   }
 })
 
+server.post('/activity_email', jsonParser, function(req, res) {    
+  var email = new sendgrid.Email({
+    to:       req.body.users,
+    from:     'wolontariat@krakow2016.com',
+    subject:  req.body.subject,
+    text:     req.body.text
+  })
+  sendgrid.send(email, function(err, json) {
+    console.log('sendgrid:', err, json)
+    res.send(err || json)
+  })  
+})
```

Comment:
Tak, to będzie po stronie serwera. Zastanawiam się czy pliki serwisów to najlepsze miejsce do umieszczania takiego kodu. Prawdopodobnie nie, ale muszę to jeszcze przemyśleć ;) Na razie możesz to tam jednak przenieść, najwyżej ja to później jeszcze raz zmienię.
Zdarzenia o których piszesz wywołują się po stronie klienta i są powiązane z interfacem.